### ISSUE TITLE
discord

### ISSUE THREAD

**Issue by binarygeek119** [DEFENDORe/pseudotv#90]
can you guys add a permanent invite link to the pseudotv on the readme.md page

i would love to join

**Comment by vexorian** [comment]
Good idea. Better yet, I will add it to the web ui. The problem right now is that no one is around to merge PRs, so changes are a bit delayed.

https://discord.com/invite/D9ckKg4
